app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt

```diff
@@ -53,6 +54,7 @@ class CallNotificationManager @Inject constructor(
     private val notificationManager = NotificationManagerCompat.from(context)
     private val scope = CoroutineScope(SupervisorJob() + dispatcherProvider.default())
     private val incomingCallsForUsers = MutableStateFlow<List<Pair<UserId, Call>>>(emptyList())
+    private val outgoingCallForUsers = MutableStateFlow<List<Pair<UserId, Call>>>(emptyList())
```

Comment:
```suggestion
    private val outgoingCallForUsers = MutableStateFlow<Map<UserId, List<Call>>>(emptyMap())
```

Question: Can the structure be like this 👆 ?
I think we want to represent the user(s) logged in with their call(s) rigth.